torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,673 @@
+        row = row.to_dict(orient="records")[0]
+        label = "None"
+        if row["properties.label"]:
+            label = row["properties.label"]
+        elif row["properties.is_crop"] == 1:
+            label = "Other"
+
+        return torch.tensor(self.classes.index(label))
+
+    def _verify(self) -> None:
+        """Verify the integrity of the dataset.
+
+        Raises:
+            DatasetNotFoundError: If dataset is not found and *download* is False.
+        """
+        # Check if feature files already exist
+        feature_path = os.path.join(
+            self.root, self.file_dict["features"]["extracted_filename"]
+        )
+        feature_path_zip = os.path.join(
+            self.root, self.file_dict["features"]["filename"]
+        )
+        label_path = os.path.join(
+            self.root, self.file_dict["labels"]["extracted_filename"]
+        )
+        # Check if labels exist
+        if os.path.exists(label_path):
+            # Check if features exist
+            if os.path.exists(feature_path):
+                return
+            # Check if features are downloaded in zip format
+            if os.path.exists(feature_path_zip):
+                self._extract()
+                return
+
+        # Check if the user requested to download the dataset
+        if not self.download:
+            raise DatasetNotFoundError(self)
+
+        # Download and extract the dataset
+        self._download()
+        self._extract()
+
+    def _download(self) -> None:
+        """Download the dataset and extract it."""
+        features_path = os.path.join(self.file_dict["features"]["filename"])
+        download_url(
+            self.file_dict["features"]["url"],
+            self.root,
+            filename=features_path,
+            md5=self.file_dict["features"]["md5"] if self.checksum else None,
+        )
+
+        download_url(
+            self.file_dict["labels"]["url"],
+            self.root,
+            filename=os.path.join(self.file_dict["labels"]["filename"]),
+            md5=self.file_dict["labels"]["md5"] if self.checksum else None,
+        )
+
+    def _extract(self) -> None:
+        """Extract the dataset."""
+        features_path = os.path.join(self.root, self.file_dict["features"]["filename"])
+        extract_archive(features_path)
+
+    def plot(self, sample: dict[str, Tensor], subtitle: Optional[str] = None) -> Figure:
+        """Plot a sample from the dataset using bands for Agriculture RGB composite.
+
+        Args:
+            sample: a sample returned by :meth:`__getitem__`
+            suptitle: optional subtitle to use for figure
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+        """
+        fig, axs = plt.subplots()
+        bands = [self.all_bands.index(band) for band in self.rgb_bands]
+        rgb = np.array(sample["array"])[:, bands]
+        normalized = rgb / np.max(rgb, axis=1, keepdims=True)
+        axs.imshow(normalized[None, ...])
```

Comment:
The example plot doesn't look very "wheat" colored. Maybe the normalization is changing the color and we should remove it? Also, we can probably remove at least the y-axis, it doesn't have any meaning. Might be good to add an x-axis label saying that this is the month, and count from 1–12 instead of 0–11.